### DISCUSSION TITLE
unable to push

### DISCUSSION THREAD

**Discussion by nicholustintzaw** [nutriverse/nutricheckr#8] [Q&A] [answered]
I am not able to push the changes made in dev_ncl branch to the remote repo. Every time I made push commend, got the error message as below. 

```
remote: error: GH006: Protected branch update failed for refs/heads/dev_ncl.        
remote: error: At least 1 approving review is required by reviewers with write access.        
To https://github.com/nutriverse/nutricheckr.git
 ! [remote rejected] HEAD -> dev_ncl (protected branch hook declined)
error: failed to push some refs to 'https://github.com/nutriverse/nutricheckr.git'
```

**Comment by ernestguevarra** [comment, marked as answer]
I have enforced branch protection on this repo such that no changes are committed to master without at least one reviewer. This means that to be able to properly interact with this repo, you will have to fork it, make changes to your fork and then make a pull request from your fork to this repo.

See this for more reading on fork vs clone - https://happygitwithr.com/fork-and-clone.html

**Reply by ernestguevarra** [reply]
This has now been resolved with @nicholustintzaw during our call today, 16 July 2021
